updateLocalesHelper/updateLocaleStatus.py

```diff
@@ -1,59 +1,92 @@
 #! /bin/env python
-import json
+from configparser import ConfigParser
+from http import HTTPStatus
+from json import dump
+from os import path, environ
```

Comment:
Same here, keeping the context by using `os.environ' seems widespread to me: https://stackoverflow.com/questions/4906977/how-to-access-environment-variable-values